util/chunk/chunk.go

```diff
@@ -277,6 +278,65 @@ func (c *Chunk) AppendPartialRow(colIdx int, row Row) {
 	}
 }
 
+// PreAlloc4Row pre-allocates the memory space for a Row.
+// Nothing except for the nullBitMap of c.columns will be pre-written.
+func (c *Chunk) PreAlloc4Row(row Row) {
+	for i, srcCol := range row.c.columns {
+		dstCol := c.columns[i]
+		dstCol.appendNullBitmap(!srcCol.isNull(row.idx))
```

Comment:
We write the nullBitMap info here is because that,
`Insert` will be called parallelly later, every element in nullBitMap is a byte .
It will cause data race problem if we write the bit mask in `Insert`.